src/app.js

```diff
@@ -0,0 +1,134 @@
+import axios from 'axios';
+
+// kritieke vragen
+// 1. valuta check werkt niet bij minder dan 2? Teller werkt niet bij 1 valuta
+// 2. pagina verversen/resultaat opschonen 2de keer zoeken?
+
+
+let fieldValue = '';
+
+// html locaties
+const imageWrapper = document.getElementById('image-wrapper')
+const nameAndFlag = document.getElementById('name-and-flag')
+const searchContainer = document.getElementById('search-container')
+const inputForm = document.getElementById("user-search-bar")
+
+// event listeners
+inputForm.addEventListener('keyup', dataField)
+// inputForm.addEventListener('keyup', cleanCheck)
+
+const countryError = document.createElement("P");
+
+
+// function cleanCheck(e) {
+//     if (e.target.value === '') {
+//         countryError.textContent = '';
+//         searchContainer.textContent = '';
+//         nameAndFlag.textContent = '';
+//         imageWrapper.removeAttribute('class')
+//         searchContainer.removeAttribute('class')
+//     }
+// }
+
+function dataField(e) {
+    fieldValue = e.target.value;
+}
+
+console.log(fieldValue)
+
+const submitSearch = document.getElementById('submit-form');
+submitSearch.addEventListener("submit", handleSubmit);
+function handleSubmit(e){
+    e.preventDefault()
+    getCountries(fieldValue);
+
+    inputForm.value = '';
+    console.log(e)
+}
+
+// fieldValue.map(() => {
+//     console.log(fieldValue)
+// })
+
+// submitSearch.addEventListener("submit", refresh)
+//
+// function refresh (e) {
+//     if ( e === 2 ) {
+//         location.reload();
+//     }
+// }
+
+const getCountries = async ( searchString ) => {
+
+    imageWrapper.textContent = '';
+    // nameAndFlag.textContent = '';
+    searchContainer.textContent = '';
+    inputForm.textContent = '';
+
+    try {
+    const response = await axios.get(`https://restcountries.com/v2/name/${searchString}`);
+
+    console.log(response)
+
+
+        //functies die content injecteren op de webpagina
+        const countryName= document.createElement("h2")
+        countryName.textContent = response.data[0].name
+        nameAndFlag.appendChild(countryName);
+
+        const countryInfo = document.createElement('p')
+        const {name, subregion, population} = response.data[0]
```

Comment:
Ik zou hier nog meer dingen als `flag` en `currencies` exposen. Eigenlijk overal waar `response.data[0].` vóor staat is een onnodige dubbeling